server.js

```
@@ -391,7 +390,11 @@ mkdirMountInfo(info, function(error)
   {
     if(error && error.code != 'EEXIST') return callback(error)
 
-    fs.symlinkSync('/proc/mounts', '/etc/mtab')
+    try {
+      fs.symlinkSync('/proc/mounts', '/etc/mtab')
+    } catch (e) {
+      // just swallow the exception
```

Please do a check similar to the one some lines before, so we can swallow only the exception about the destination exception already exists and capture the other ones.